prow/scripts/cluster-integration/serverless-git-auth-integration-nightly-k3s.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```

Comment:
Can we use one script `serverless-integration-k3s.sh` and pass parameter: e.g.: `job_name="k3s-serverless-nightly-test"?

Reply:
Initially I just added a few if conditions tot eh original script, but it got a bit ugly. So, I moved shared code an external file and split the tests.

Reply:
Is there any difference (except env) running gitauth test and integration-tests?